gitlab/v4/objects/merge_requests.py

```diff
@@ -373,6 +389,7 @@ class ProjectMergeRequestManager(CRUDMixin, RESTManager):
             "remove_source_branch",
             "allow_maintainer_to_push",
             "squash",
+            "reviewer_ids",
```

Comment:
As long as we are updating this I think it would be good to bring it up to date with the current API

https://docs.gitlab.com/ee/api/merge_requests.html#create-mr

These seem missing to me:

- assignee_ids | integer array | no | The ID of the user(s) to assign the MR to. Set to 0 or provide an empty value to unassign all assignees.
- reviewer_ids | integer array | no | The ID of the user(s) added as a reviewer to the MR. If set to 0 or left empty, no reviewers are added.
- remove_source_branch | boolean | no | Flag indicating if a merge request should remove the source branch when merging.

Reply:
Thanks for taking a look at this.

Yeah, what's interesting about this particular API is that, unlike most other endpoints, the POST body includes parameters that are _not_ included in the response.

One consequence I noticed of adding a name to the list is that the framework will try to make that name an accessible attribute on the RestObject. However, because `reviewer_ids` is not present in the response, you get a (arguably unexpected) error when trying to access it. 

This is not a _huge_ problem, but was a motivation behind the `@property` getter/setter for `reviewer_ids` in an attempt to keep the Python API more consistent.

If the idea is to be able to do something like this:
```python
mr.reviewer_ids = [1,2,3]
mr.save()
```

Then `mr.reviewer_ids` should always be an accessible attribute even on a freshly retrieved object, in my opinion (without the property, this will be an attribute error because it's not in the response as such)

If this is seen as a good approach, I'd be happy to add the same for assignees (it has the same oddities, IIRC).

I'll also be happy to take care of the chore of adding `remove_source_branch` and double-checking for any other missing parameters for MRs.